### ISSUE TITLE
Trees grown using tree fertilizer are not rendered

### ISSUE THREAD

**Issue by dj-bauer** [hpeinar/stardewplanner#64]
I have a row of trees of which all except three are grown using tree fertilizer.
Unfortunatly only the single three ones are actually rendered.
![1NMEzU-plan](https://user-images.githubusercontent.com/31740488/163992831-236c3f00-6ee7-4c9f-9911-6f9a0adca4a6.png)

Playing on a 1.5.6 world using (obviously) forest layout.
I can provide more information if necessary.

**Comment by hpeinar** [comment]
Yeah, there are bunch of missing asset linking between the upload.farm renderer and planner.

You could try loading your farm in [https://v3.stardew.info/](https://v3.stardew.info) and see what does it looks like there. Just append your farm ID to the v3 planner URL.

**Comment by dj-bauer** [comment]
That sounds like a good approach.
Is the sourcecode for the new version in any repo or branch available?
I would like to try to contribute.

**Comment by hpeinar** [comment]
At the moment there are no plans to make the new version of the planner open source as it is using Stardew Valley assets for which I don't have the rights to include in a open source project.
